OrbitCore/TracepointEventBuffer.cpp

```diff
@@ -61,7 +62,11 @@ void TracepointEventBuffer::ForEachTracepointEventOfThreadInTimeRange(
     int32_t thread_id, uint64_t min_tick, uint64_t max_tick,
     const std::function<void(const orbit_client_protos::TracepointEventInfo&)>& action) const {
   ScopeLock lock(mutex_);
-  if (thread_id == SamplingProfiler::kAllThreadsFakeTid) {
+  if (thread_id == SamplingProfiler::kAllTracepointsFakeTid) {
+    for (const auto& entry : tracepoint_events_) {
+      ForEachTracepointEventInEventMapInTimeRange(min_tick, max_tick, entry.second, action);
```

Comment:
The logic behind the name is that the function performs an action (any action you want since you pass that as a parameter) for each tracepoint event in an event map in a time range. @dpallotti suggested me this name but if you have any idea feel free to tell. To come with a name expressing what the function is doing to the passed element is a bit tricky since the action can be any action you want.